platform/android/display_server_android.cpp

```diff
@@ -538,9 +538,9 @@ void DisplayServerAndroid::reset_window() {
 #endif
 
 		if (context_rd->window_create(MAIN_WINDOW_ID, last_vsync_mode, display_size.width, display_size.height, &wpd) != OK) {
+			ERR_FAIL_MSG(vformat("Failed to reset %s window.", context_rd->get_api_name()));
 			memdelete(context_rd);
 			context_rd = nullptr;
```

Comment:
`ERR_FAIL_MSG` returns, so that's a memory leak.

Should be `ERR_PRINT` and then a return at the end.

Reply:
Good catch! Otherwise, this PR was trading one kind of error for another. Done.